src/index.js

```diff
@@ -3,4 +3,14 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import registerServiceWorker from './registerServiceWorker';
 
+const App = () => {
+    return (
```

Comment:
Remember, your JSX in your return statement can only have 1 top-level element.  In this case you have 3.  So, you'll want to wrap your <h1>, <p>, and <h2> tags inside of a <div>.

We'll be going over this again next week, so you'll feel really comfortable making React components :)